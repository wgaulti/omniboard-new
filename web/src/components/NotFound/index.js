import React, {Component} from 'react';
import PropTypes from 'prop-types';
import classnames from 'classnames';

import './style.scss';

export default class NotFound extends Component {
  static propTypes = {
    className: PropTypes.string,
    staticContext: PropTypes.array
  };

  render() {
    const {className, staticContext: _ignored, ...props} = this.props;
    return (
      <div className={classnames('NotFound', className)} {...props}>
        <h1>
          404 <small>Not Found :(</small>
        </h1>
      </div>
    );
  }
}
