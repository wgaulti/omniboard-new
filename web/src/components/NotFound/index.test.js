import React from 'react';
import NotFound from '.';

describe('NotFound', () => {
  it('should render', () => {
    const wrapper = shallow(<NotFound/>);

    expect(wrapper).toMatchSnapshot();
  });
});
