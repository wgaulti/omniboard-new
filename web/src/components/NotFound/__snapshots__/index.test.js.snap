// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render 1`] = `
<div
  className="NotFound"
>
  <h1>
    404 
    <small>
      Not Found :(
    </small>
  </h1>
</div>
`;
